### ISSUE TITLE
DB Integration for Create/Read/Update Question

### ISSUE THREAD

**Issue by ChristopherDavenport** [EckerdCollege/alterpass#53]
Table Required: ____
Query: ___

**Comment by djkcel** [comment]
Is this a Walter/Frank thing?

**Comment by ChristopherDavenport** [comment]
Yes. Frank was. This is necessary information for stage 2 involving questions. Although i can write my own, it's likely you want a more robust solution involving banner data.

**Comment by abneyfl** [comment]
It has been on my list for a while, but I am not getting to much of my
list.  Not sure when I will be able to do this piece.  I will talk to
Walter and Kevin about options, but Walter is out this week.

Frank Abney
Assistant Director ITS
Eckerd College
